Eradication/LarvalHabitatMultiplier.cpp

```diff
@@ -75,7 +75,19 @@ namespace Kernel
 
     float LarvalHabitatMultiplier::GetMultiplier( VectorHabitatType::Enum vht, const std::string& species ) const
     {
-        return m_Multiplier.at( vht ).at( species );
+        if( !m_Multiplier.count( vht ) )
+        {
+            release_assert( false );
+        }
+
+        if( !m_Multiplier.at( vht ).count( species ) )
+        {
+            release_assert( false );
+        }
+        else
+        {
+            return m_Multiplier.at( vht ).at( species );
+        }
```

Comment:
Did you always have this method this way?  I would swear you had line 85 returning 1.  However, I like this.  You could make it cleaner and look something like:

release_assert( m_Multiplier.count( vht ) > 0 );
release_assert( m_Multiplier.at( vht ).count( species ) > 0 );
return m_Multiplier.at( vht ).at( species );

Reply:
it was returning 1 as the "fix" for the m_Multiplier being initialized incorrectly, but I changed it so now it's a release assert if it gets here and it's not initialized properly.